apps/src/templates/studioHomepages/SectionsTable.jsx

```diff
@@ -89,36 +100,56 @@ const SectionsTable = React.createClass({
         sectionCode: React.PropTypes.string.isRequired
       })
     ),
-    isRtl: React.PropTypes.bool.isRequired
+    isRtl: React.PropTypes.bool.isRequired,
+    isTeacher: PropTypes.bool.isRequired,
+  },
+
+  onLeave() {
+    console.log("How do I remove a student from a section?");
   },
```

Comment:
I don't intend to ship with this console.log; it's just here until we have a definitive decree from PMs on how to proceed.